O365/message.py

```diff
@@ -755,7 +756,20 @@ def reply(self, to_all=True):
             return None
 
         message = response.json()
+        message_body_content_html  = message['body']['content']
 
+        pattern = re.compile(r'<b>Sent:</b> (.*?)<br>')
+
+        def change_sent_time(match):
+            sent_time_str = match.group(1)
+            sent_time = dt.datetime.strptime(sent_time_str, '%A, %B %d, %Y %I:%M:%S %p')
+            sent_time_new = sent_time.replace(tzinfo=dt.timezone.utc).astimezone(self.protocol.timezone)
+            return '<b>Sent:</b> {:%A, %B %d, %Y %I:%M:%S %p}<br>'.format(sent_time_new)
+
+        modified_data = pattern.sub(change_sent_time, message_body_content_html )
+
+        message['body']['content'] = modified_data
```

Comment:
Per my previous comment, we can delete the above line and combine the two lines into the following:
```
# Inject original message body content's Sent time with a timezone
message['body']['content'] = pattern.sub(change_sent_time_timezone, message_body_content_html)
```